Makefile

```diff
@@ -20,16 +20,23 @@ coverage:
 docs:
 	bundle exec rdoc lib -o docs --title "EasyPost Ruby Docs"
 
-## fix - Fix Rubocop errors
-fix:
+## format - Fix Rubocop errors
+format:
 	bundle exec rubocop -a
 
+## install-styleguide - Import the style guides (Unix only)
+install-styleguide: | update-examples-submodule
+	sh examples/symlink_directory_files.sh examples/style_guides/ruby .
+
 ## install - Install globally from source
-install:
-	git submodule init
-	git submodule update
+install: | update-examples-submodule
```

Comment:
question: shouldn't install also be installing all these styleguides? I'd expect install to setup the whole project for me.

This is true for all the ones we've already done.

Reply:
If `make install` triggered `make install-styleguide` (a Unix-only command), no Windows user would be able to run `make install`. `make install-styleguide` has been made a standalone, manual step for this reason.

Reply:
Make is already a unix only tool 😉

Reply:
Fair, though all I can say is this will frustrate people (us), because we'll forget this is a thing.

Reply:
Well, all you have to do is run it once and the symlink will be there forever :)